src/main.py

```diff
@@ -168,6 +174,15 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
     # Noise on windows at the end of the logs is caused by https://github.com/aio-libs/aiohttp/issues/4324
 
 
+async def check_if_project_is_disabled_and_get_disabled_api_set(context: MetricsContext, project_id: str) -> [str, bool, set]:
```

Comment:
I believe we are returning "Tuple[str, bool, Set[...]]", aren't we?
https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

Reply:
I think it's the same, in python using a comma is basically defining a tuple?

Reply:
Yes. And [] would be a list

Reply:
It changes nothing, but type hints are simply that, hints